errors/Readme.md

```diff
@@ -155,13 +160,61 @@ If we encounter a Rust panic, that is covered in the [Unknown Errors](#unknown-e
 
 SDK acts as the interface between the binaries and the tools. This section covers errors from SDK, binaries and the network between SDK ⇆ Binary and Binary ⇆ Data source.
 
-This spec does not list the individual errors. Instead the single source of truth for them is our code where they are defined with their codes and messages. Here is a list of links to their defintions for each error category:
+This spec does not list individual errors (Except for some errors in the "common" category which occur in the Javascript part of the code, these errors are listed below). Instead, the single source of truth for them is our code where they are defined with their codes and messages. Here is a list of links to their definitions for each error category:
 
 - [Common](https://github.com/prisma/prisma-engine/blob/master/libs/user-facing-errors/src/common.rs)
 - [Query Engine](https://github.com/prisma/prisma-engine/blob/master/libs/user-facing-errors/src/query_engine.rs)
 - [Migration Engine](https://github.com/prisma/prisma-engine/blob/master/libs/user-facing-errors/src/migration_engine.rs)
 - [Introspection Engine](https://github.com/prisma/prisma-engine/blob/master/libs/user-facing-errors/src/introspection_engine.rs)
 
+### Common
+
+The following errors have to be listed separately because they happen in the Javascript code that wraps the rust binaries.
+
+#### P1004: Incompatible binary
+
+- **Description**: The downloaded/provided binary `${binary_path}` is not compiled for platform `${platform}`
+- **Meta schema**:
+  ```ts
+  type Meta = {
+    // Fully resolved path of the binary file
+    binary_path: string
+    // Identifiers for the currently identified execution environment, e.g. `native`, `windows`, `darwin` etc
+    platform: string
+  }
+  ```
+
+#### P1005: Unable to start the query engine
+
+- **Description**: Failed to spawn the binary `${binary_path}` process for platform `${platform}`
+- **Meta schema**:
+  ```ts
+  type Meta = {
+    // Fully resolved path of the binary file
+    binary_path: string
+    // Identifiers for the currently identified execution environment, e.g. `native`, `windows`, `darwin` etc
+    platform: string
+  }
+  ```
+
+#### P1006: Binary not found
+
+- **Description**: Photon binary for current platform `${platform}` could not be found. Make sure to adjust the generator configuration in the `schema.prisma` file. <br /> <br />`${generator_config}` <br /> <br />Please run `prisma2 generate` for your changes to take effect.
```

Comment:
Photon is not a thing any more

Reply:
Updated it to be called "Query engine binary"